packages/authentication/src/definitions.ts

```diff
@@ -85,6 +93,14 @@ export interface FirebaseAuthenticationPlugin {
    * @since 0.2.2
    */
   sendPasswordResetEmail(options: SendPasswordResetEmailOptions): Promise<void>;
+  /**
+   * Sends a sign-in email link to the user with the specified email.
+   *
+   * To complete sign in with the email link, call signInWithEmailLink with the email address and the email link supplied in the email sent to the user.
+   *
+   * @since 0.6.0
+   */
+  sendSignInLinkToEmail(options: SignInWithEmailLinkOptions): Promise<void>;
```

Comment:
Please name the options interface like the method and append `Options` to it: `SendSignInLinkToEmailOptions`